.gitignore

```diff
@@ -16,8 +16,7 @@ bin/
 
 # Binaries are copied (as needed) to the same location as their respective Dockerfile in
 # order to simplify docker build commands. Ignore these files if proper clean up fails.
-cmd/occollector/occollector_linux
-cmd/otelsvc/otelsvc_linux
+cmd/otelcol/otelcol_linux
```

Comment:
nit: we don't build `_$(GOOS)` anymore, the possible left over in this case is cmd/otelcol/otelcol

Reply:
Fixed.